pkg/messaging/v1/util.go

```diff
@@ -0,0 +1,211 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package v1
+
+import (
+	"net/http"
+	"strings"
+
+	commonv1pb "github.com/dapr/dapr/pkg/proto/common/v1"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/any"
+	structpb "github.com/golang/protobuf/ptypes/struct"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	// JSONContentType is the MIME media type for JSON
+	JSONContentType = "application/json"
+	// ProtobufContentType is the MIME media type for Protobuf
+	ProtobufContentType = "application/x-protobuf"
+
+	// DaprHeaderPrefix is the prefix if metadata is defined by non user-defined http headers
+	DaprHeaderPrefix = "dapr-"
+
+	// gRPCBinaryMetadata is the suffix of grpc metadata binary value
+	gRPCBinaryMetadata = "-bin"
```

Comment:
nit, rename to gRPCBinaryMetadataSuffix

Reply:
Good catch. Ok. I will rename it to `gRPCBinaryMetadataSuffix` in the next PR.